content/posts/2023-04-05-microshift-on-crc.md

```diff
@@ -0,0 +1,35 @@
+---
+title: " Introducing MicroShift preset for CRC"
+date: 2023-04-05T07:12:10+05:30
+---
+
+Are you a developer who wants to use OpenShift Kubernetes Engine but doesn't have a powerful system?
+Or do you need to deploy to a small form factor or edge computing environment? 
+If so, you'll be happy to hear about our new [MicroShift](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12/html/getting_started/index) preset,
+a lightweight version of OpenShift Kubernetes Engine that's optimized for resource-limited environments.
+
+The OpenShift preset currently provided by CRC requires 9GB of RAM and 4 CPUs.
+With the release of CRC 2.16, you can now use the MicroShift preset, which only requires 4GB of RAM and 2 CPUs.
+
+What makes MicroShift different from OpenShift Kubernetes Engine? MicroShift is designed for small form factor
+and edge computing environments, which means it's optimized for lower resource requirements. MicroShift
+uses a subset of the OpenShift Kubernetes Engine components and removes some of the features that aren't needed in these environments.
+
+[Key differences from OpenShift Kubernetes Engine](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12/html/getting_started/microshift-architecture#microshift-differences-oke_microshift-architecture)
+
+To get started with MicroShift, you'll need to download and install the [latest version](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/crc/latest) of CRC.
+If you already have a CRC instance, delete it using the `crc delete` command.
+Then set the preset to MicroShift using the `crc config set preset microshift` command.
+Finally, set up and start CRC using the `crc setup` and `crc start` commands.
+In case you want to use `crc oc-env` command, do check the [workaround](https://github.com/crc-org/crc/issues/3581#issuecomment-1497174956)
```

Comment:
Maybe "In order to use `oc` and `crc oc-env` with crc version 2.16, a [workaround] is needed" ?

Reply:
> In order to use oc

Not `oc`, which works without any issue if user have it in the path. it is just the `oc-env` which have problem for creating the symlink to path.

Reply:
Maybe "In order to use crc oc-env with crc version 2.16, a [workaround] is needed" ?